grails-app/views/consumption/show.gsp

```diff
@@ -26,7 +26,7 @@
 
                 <div class="box">
                     <h2>
-                        ${warehouse.message(code:'consumption.label')} <small>(${command.rows?.keySet()?.size()} results)</small>
+                        ${warehouse.message(code:'consumption.label')} <small>(${command.rows?.keySet()?.size()} ${g.message(code: 'default.resultsLowerCase.label', default: 'results')})</small>
```

Comment:
In the future, let's not add codes for lowercase vs uppercase. You should be able to the following
```
${g.message(code: 'default.resultsLowerCase.label', default: 'results').toLowerCase()}
```
or 
```
<%@ page import="import org.apache.commons.lang3.StringUtils" %>
${StringUtils.lowerCase(g.message(code: 'default.results.label', default: 'Results'))}
```